packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -57,6 +57,8 @@ export default deepFreeze(
         },
       ],
       search: {
+        servicePolling: true,
+        servicePollingInterval: 5000,
```

Comment:
should this be defaulted to enabled? I would think that this should be opt-in.

Reply:
although I suppose the main use case would be when developing, where a config is not available 🤔